dba/dba.php

```diff
@@ -372,6 +372,6 @@ function dba_list(): array {}
  * value_name). This function will return <b>FALSE</b> if
  * <i>key</i> is <b>NULL</b> or <b>FALSE</b>.
  */
-function dba_key_split(string|false|null $key): array|false {}
+function dba_key_split(#[LanguageLevelTypeAware(["8.4" => "string"], default: "string|false|null")] $key): array|false {}
```

Comment:
Should this be changed? Could you please share a link on RFC or PHP change?

Reply:
Such deprecation can't be configured with different typehints in sugnature. Please see https://youtrack.jetbrains.com/issue/WI-79525/Deprecations-for-PHP-8.4 . A separate inspection should be done for this